src/main/java/io/fabric8/jenkins/openshiftsync/OpenShiftUtils.java

```diff
@@ -144,6 +144,15 @@ public static boolean isJenkinsBuildConfig(BuildConfig bc) {
         return false;
     }
 
+    public static boolean isJenkinsBuild(Build b) {
```

Comment:
isPipelineStrategyBuild would be a better name i think.

Reply:
yeah I went with consistency with the analogous method for BC's ... but we are reworking stuff and renaming is easy with eclipse .... making the changes